src/server/command_registry.cc

```diff
@@ -181,9 +203,19 @@ CommandRegistry::CommandRegistry() {
 }
 
 void CommandRegistry::Init(unsigned int thread_count) {
+  const auto alias_mappings = AliasMap();
```

Comment:
please use CmdLineMapping instead of auto
Also I would rename the 
alias_mappings -> origing_to_alias_cmd
and AliasMap -> GetOriginToAliasMap

Reply:
fixed